src/NUnitEngine/nunit.engine/Services/DomainManager.cs

```diff
@@ -194,12 +192,13 @@ private void UnloadOnThread()
                 }
                 catch (Exception ex)
                 {
-                    _unloadException = ex;
-
                     // We assume that the tests did something bad and just leave
-                    // the orphaned AppDomain "out there". 
-                    // TODO: Something useful.
-                    log.Error("Unable to unload AppDomain " + domainName, ex);
+                    // the orphaned AppDomain "out there".
+                    var msg = DomainDetailsBuilder.DetailsFor(_domain,
+                    $"Exception encountered unloading AppDomain: {ex.Message}");
```

Comment:
Either indent this line or extract `$"Exception encountered unloading AppDomain: {ex.Message}"` to a variable and keep `var msg = DomainDetailsBuilder.DetailsFor(_domain, errMsg);` on one line